### DISCUSSION TITLE
Make really small button

### DISCUSSION THREAD

**Discussion by UlrichGoebel** [Textualize/textual#4304] [General]
I'm a textual beginner, so my question will be a problem of beginners...

I would like to build really small buttons: just one character (like ✓ ⮽ ↻ ↺ 🖫 🔍) plus one blank on left and right, nothing else.

What I tried:

```
#!/usr/bin/env python3

from textual.app import App
from textual.widgets import Button, Static

class main(App):
    
    CSS = """
        #quit-button {
            width: 3;
            height: 1;
            padding: 0;
            text-align: center;
            }
        
        #wanted {
            width: 3;
            height: 1;
            text-align: center;
            background: $primary;
            color: auto;
            }
    """
    
    def compose(self):
        yield Button('Q', id='quit-button', variant='warning')
        yield Static('Q', id='wanted')
        
if __name__ == '__main__':
    app = main()
    app.run()
```

The `Static` (with style `wanted`) shows what I want, the Button (with style `quit-button`) doesn't do what I hope. The `width` there doesn't have any effect, the `height` makes the "Q" invisible; `with` and `height` make the button large, if I set them to 50 and 5. There seems to be a minimal size of Buttons

Can somebody help me?

Thanks
Ulrich

**Comment by davep** [comment]
You may find [the note about styling buttons](https://textual.textualize.io/widgets/button/#additional-notes) helpful.

**Reply by UlrichGoebel** [reply]
Wow, that came very fast, thank you very much!
And it solved my problem. Thank's again.

**Reply by davep** [reply]
Most welcome; glad it helped. :-)
